docs/src/examples/gallery.md

```diff
@@ -377,3 +377,99 @@ savefigs("patch-inline", ans) # hide
 [\[.pdf\]](patch-inline.pdf), [\[generated .tex\]](patch-inline.tex)
 
 ![](patch-inline.svg)
+
+-----------------------
+
+## 3D Waterfall
+
+```@example pgf
+using Random
+using Distributions
+
+Random.seed!(42)
+#Generate Data
+x_min = -10 # xrange to plot
+x_max = 10
+μ_min = -5
+μ_max = 5
+
+dist = (μ, σ) ->  Normal(μ, σ)      # use normal distributions
+dists = [dist(-6+i, 1+0.3*i) for i in 1:10]       # make the set of distributions we're going to plot
+rnd = rand.(Truncated.(dists, x_min, x_max),20)  # creates random scatter points
+dat_pdf = [(x) ->  pdf(d,x) for d in dists ] # Get the pdf of the dists
```

Comment:
```suggestion
dat_pdf = [(x) -> pdf(d, x) for d in dists] # Get the pdf of the dists
```
Also, while a closure is fine, couldn't you just call `pdf.(dists[i], x_pnts)` in the code below?